app/i18n/en/footer.ts

```diff
@@ -4,86 +4,170 @@ const footer: Footer = {
+        },
+      ],
     },
     {
       question: 'Direct Marketing and Do Not Track',
-      answer:
-        'Virufy does not track its customers over time and across third party websites to provide targeted advertising and therefore does not respond to web browsers “do not track” signals. If you arrive at our website by way of a link from a third party site that does respond to “do not track” requests, the recognition of any “do not track” request you have initiated will end as soon as you reach our website. Third parties cannot collect any other personally identifiable information from Virufy’s websites unless you provide it to them directly. This is also described in our Privacy Policy.',
+      answer: [
+        {
+          type: 'text',
+          text: 'Virufy does not track its customers over time and across third party websites to provide targeted advertising and therefore does not respond to web browsers “do not track” signals. If you arrive at our website by way of a link from a third party site that does respond to “do not track” requests, the recognition of any “do not track” request you have initiated will end as soon as you reach our website. Third parties cannot collect any other personally identifiable information from Virufy’s websites unless you provide it to them directly. This is also described in our Privacy Policy.',
+        },
+      ],
     },
     {
       question: 'Notice for Minors',
-      answer:
-        'We do not knowingly collect online information from children under the age of 18. If we are notified that we have collected personal information, as defined by the Children’s Online Privacy Protection Act (“COPPA”), of a child under the age of 13, we will delete the information as expeditiously as possible. We never sell the personal information of minors under 18 years of age and would not do so in the future without affirmative authorization of the consumer if between 13 to 16 years of age, or the parent or guardian of a consumer less than 13 years of age.',
+      answer: [
+        {
+          type: 'text',
+          text: 'We do not knowingly collect online information from children under the age of 18. If we are notified that we have collected personal information, as defined by the Children’s Online Privacy Protection Act (“COPPA”), of a child under the age of 13, we will delete the information as expeditiously as possible. We never sell the personal information of minors under 18 years of age and would not do so in the future without affirmative authorization of the consumer if between 13 to 16 years of age, or the parent or guardian of a consumer less than 13 years of age.',
+        },
+      ],
     },
   ],
   sectionPrivacyPolicy: [
     {
       question: 'Argentina',
-      answer: 'Política de Privacidad',
-      link: 'https://drive.google.com/file/d/1Fef7l77QP3EZboej6mw_q7fBNrayvs03/view',
+      answer: [
+        {
+          type: 'text',
```

Comment:
this needs to be 'link'

Reply:
nice catch. Resolved in most recent commit.